src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -169,6 +171,13 @@ export default class DataTable extends Vue {
     return null;
   }
 
+  _bulkAction() {
+    if (this.$scopedSlots['bulkActions']) {
+      return <div class="">{this.$scopedSlots['bulkActions']({})}</div>;
```

Comment:
Any reason there is an empty class here?

Reply:
No specific reason. I have referred to existing toolbar method. I can remove it.